server/routes.js

```diff
@@ -35,11 +34,12 @@ module.exports = (app) => {
    */
   app.use(robots(path.join(__dirname, '../frontend/dist/robots.txt')));
 
-  /**
-   * Authentication
-   */
-  app.get('/api/auth/:service(github)/:slug', aN.authenticateService);
-  app.get('/auth/:service(github)/callback/:slug', aN.authenticateServiceCallback);
+  // For some reason during OAuth, the redirect through piping automatically changes 
+  // the body, but not the window location -> instead redirect manually
```

Comment:
I've never done redirects with piping, I found a SO question about it

http://stackoverflow.com/questions/13954094/redirect-express-js-piped-req-pipe-res-properly